pytest_fixtures/broker.py

```diff
@@ -168,3 +168,10 @@ def registered_hosts(organization_ak_setup, content_hosts):
         vm.register_contenthost(org.label, ak.name)
         assert vm.subscribed
     return content_hosts
+
+
+@pytest.fixture
+def destructive_sat(satellite_latest):
+    """Destructive tests require changing settings.server.hostname for now"""
+    with satellite_latest as sat:
+        yield sat
```

Comment:
@JacobCallahan If we use this for each destructive test then that will increase overall test execution time significantly. Are we ok with that?

Reply:
we are indeed. there will be different scopes throughout destructive tests. some will be function-scope like this, while others will be module or session scope.

Reply:
Why create a wrapper here, simply for the name? `satellite_latest` is yielding already.

Reply:
satellite_latest doesn't yield from the satellite object's context manager.